shopfloor/docs/zone_picking_diag_seq.txt

```diff
@@ -0,0 +1,43 @@
+# Diagram to generate with PlantUML (https://plantuml.com/)
+#
+#   $ sudo apt install plantuml
+#   $ plantuml zone_picking_diag_seq.txt
+#
+
+@startuml
+
+start -> select_picking_type: **/scan_location**
+select_picking_type -> select_line: **/list_move_lines**
+select_line -> set_line_destination: **/scan_source**
+set_line_destination -[#red]> set_line_destination: **/set_destination** \n(error)
+set_line_destination -> set_line_destination: **/set_destination** \n(scanned location not expected but valid, confirmation required)
+set_line_destination -> select_line: **/set_destination**
+set_line_destination -> zero_check: **/set_destination**
+zero_check -> select_line: **/is_zero**
+select_line -> unload_single: **/prepare_unload** \n(different destinations in buffer lines, process one by one)
+select_line -> unload_set_destination: **/prepare_unload** \n(only one move line in the buffer)
+select_line -> unload_all: **/prepare_unload** \n(buffer lines have all the same destination location)
+select_line -> select_line: **/prepare_unload** \n(no remaining lines in the buffer)
+unload_all -[#red]> unload_all: **/set_destination_all** \n(error, scanned destination location invalid)
+unload_all -> unload_all: **/set_destination_all** \n(scanned destination not expected but valid, confirmation required)
+unload_all -> select_line: **/set_destination_all**
```

Comment:
`unload_all` should have a return to `start` when all lines are done

Reply:
done, plus:
+ added the call to `/select_zone` when the `start` screen is opened
+ added separators between endpoints, could make things clearer